File: Hybrid_Task/Java/Dinner/GCD.java
import java.util.Scanner;

public class GCD {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

System.out.print("Enter a number: ");
int numberOne = input.nextInt();

System.out.print("Enter another number: ");
int numberTwo = input.nextInt();

int max = Math.max(numberOne, numberTwo);
int min = Math.min(numberOne, numberTwo);

int divisor = 2;
int greatest = 1;
while(divisor <= max) {
	if(max % divisor == 0 & min % divisor == 0) {
		if(divisor > greatest) greatest = divisor; }
divisor++;
}
System.out.print(greatest);
}
}

File: Hybrid_Task/Python/Dinner/gcd.py
number_one = int(input("Enter a number: "))
number_two = int(input("Enter another number: "))

max = max(number_one,number_two)
min = min(number_one,number_two)

divisor = 2 
greatest = 1

while divisor <= max:
	if max % divisor == 0 and min % divisor == 0:
		if divisor > greatest:
			greatest = divisor
		
			
	divisor += 1

print(greatest)

